.github/workflows/lint.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         with:
           cache-on-failure: true
       - run:
-          cargo clippy --bin "${{ matrix.binary }}" --workspace --features "${{ matrix.network }}"
+          cargo clippy --bin "${{ matrix.binary }}" --workspace --features "${{ matrix.network }}" --lib --tests --benches --examples
```

Comment:
--all-targets?

Reply:
we're testing --bin wich conflict due to optimism feature, and all-target contains --bins